README.md

```diff
@@ -3,6 +3,43 @@
 
 > Node module for parsing screwdriver.yaml configurations
 
+This tool works by passing the screwdriver.yaml through four phases:
```

Comment:
These four phases might need extra explanation. So I think either we need to: 1. keep it really simple here and explain elsewhere (like the guide). or 2. explain a little bit more below each phase. 

For number 2, you can move all the text in the parenthesis to below the numbers, and explain a little more about the phase